templates/logging-sidecar-configmap.yaml

```diff
@@ -57,6 +57,10 @@ data:
             .level = parsed_log.level
             .logger = parsed_log.logger
             .chan = parsed_log.chan
+            # Extract error details if present
+            if exists(parsed_log.error_detail) {
+              .error_detail = parsed_log.error_detail
+            }
```

Comment:
Instead of extracting named fields, we should be able to use something like the following:
```
parsed_log = parse_json!(.message)
. = merge!(., parsed_log)
```